dbt/include/global_project/macros/materializations/archive/archive.sql

```diff
@@ -18,12 +75,12 @@
 
         select
             {% for col in adapter.get_columns_in_table(source_relation.schema, source_relation.identifier) %}
```

Comment:
shouldn't this just be `for col in cols`?

Reply:
yes, 100%! Good catch